code/modules/mob/living/carbon/human/species/station/protean_vr/protean_blob.dm

```diff
@@ -746,8 +746,48 @@
 			I.plane = PLANE_LIGHTING_ABOVE
 			add_overlay(I)
 			qdel(I)
+		if(S.blob_appearance == "dullahan")
```

Comment:
Upstream file. do not modify without comments

Reply:
where should i put the comments?

Reply:
actually, would this just be better to send upstream?

Reply:
put some //chomp edit comments in